packages/@divvi/mobile/locales/base/translation.json

```
@@ -2638,6 +2638,13 @@
       "myPools": "My Pools",
       "explore": "Explore Open"
     },
+    "depositConfirmation": {
+      "title": "Review Deposit",
+      "depositing": "Depositing",
+      "into": "Into {{providerName, getCorrectArticle}}",
```

```suggestion
      "into": "Into",
```

You aren't gonna need it (YAGNI) applies here, and I don't think the custom formatter is worth the squeeze. We'd need to do the same for other languages in Valora so it's better to just remove a{n}.